ocaml/parsing/builtin_attributes.ml

```diff
@@ -94,11 +94,9 @@ let builtin_attrs =
   ; "flambda_o3"; "ocaml.flambda_o3"
   ; "afl_inst_ratio"; "ocaml.afl_inst_ratio"
   ; "local_opt"; "ocaml.local_opt"
-  ; "curry"; "ocaml.curry"; "extension.curry"
-  ; "global"; "ocaml.global"; "extension.global"
-  ; "local"; "ocaml.local"; "extension.local"
-  ; "nontail"; "ocaml.nontail"; "extension.nontail"
-  ; "tail"; "ocaml.tail"; "extension.tail"
+  ; "curry"; "ocaml.curry" (* For [-extension local] *)
```

Comment:
Should we remove the `extension` version of attributes in a separate change? This is a fairly large PR and I'm eager to see if there are ways to split out indepedently reviewable/rollable pieces.

Reply:
I like the impulse, but I don't see how for this one -- if we want to support `extension.foo`, we have to leave all the attribute/extension support in anyway.

Reply:
You're right, I don't see a better way of staging that.